scripts/prepare_alpaca.py

```diff
@@ -18,7 +18,7 @@
 DATA_FILE_NAME = "alpaca_data_cleaned_archive.json"
 DESTINATION_PATH = Path("data/alpaca")
 CHECKPOINT_DIR = Path("checkpoints/stabilityai/stablelm-base-alpha-3b")
-TEST_SPLIT_SIZE = 2000
+TEST_SPLIT_PERCENT = 3.865	# to get exactly 2000 test samples
```

Comment:
It's more common to represent a percentage as a fraction of 1. This will simplify the code below because you won't need to divide by 100

Also avoiding tabs

```suggestion
TEST_SPLIT_PERCENT = 0.3865  # percentage to get exactly 2000 samples
```

Reply:
Needs to be 0.03865.